src/content/docs/infrastructure/host-integrations/installation/install-infrastructure-host-integrations.mdx

```diff
@@ -40,83 +39,74 @@ import osUbuntu from 'images/os_icon_ubuntu.webp'
+Some of our infrastructure integrations work in tandem with our infrastructure agent, and have the same file and configuration structure. We sometimes refer to these as the infrastructure agent integrations. These integrations include: 
 
-To collect data from on-host services, all you need is:
-- The [infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent)
-- The [on-host integrations](/docs/integrations/host-integrations/getting-started/introduction-host-integrations).
+* [Apache](/docs/integrations/host-integrations/host-integrations-list/apache-monitoring-integration) 
+* [Cassandra](/docs/integrations/host-integrations/host-integrations-list/cassandra-monitoring-integration)
+* [Couchbase](/docs/integrations/host-integrations/host-integrations-list/couchbase-monitoring-integration)
+* [Elasticsearch](/docs/integrations/host-integrations/host-integrations-list/elasticsearch-monitoring-integration)
+* [HAProxy](/docs/integrations/host-integrations/host-integrations-list/haproxy-monitoring-integration)
+* [HashiCorp Consul](/docs/integrations/host-integrations/host-integrations-list/hashicorp-consul-monitoring-integration)
+* [JMX](/docs/integrations/host-integrations/host-integrations-list/jmx-monitoring-integration)
+* [Kafka](/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration)
+* [Memcached](/docs/integrations/host-integrations/host-integrations-list/memcached-monitoring-integration)
+* [MongoDB](/docs/integrations/host-integrations/host-integrations-list/mongodb-monitoring-integration)
+* [MySQL](/docs/integrations/host-integrations/host-integrations-list/mysql-monitoring-integration)
+* [NGINX](/docs/integrations/host-integrations/host-integrations-list/nginx-monitoring-integration)
+* [PostgreSQL](/docs/integrations/host-integrations/host-integrations-list/postgresql-monitoring-integration)
+* [RabbitMQ](/docs/integrations/host-integrations/host-integrations-list/rabbitmq-monitoring-integration) 
+* [Redis](/docs/integrations/host-integrations/host-integrations-list/redis-monitoring-integration)
 
-All on-host integrations require that our infrastructure agent is [installed on the host](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent). Besides collecting system's data, the agent acts as a forwarder for integrations's data, and can forward [logs](/docs/logs/enable-log-monitoring-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent).
+There are different ways to install these integrations, which will be explained below. For technical details about the file structure and configuration, see [Technical details](#technical-details).
 
-For example, for SQL Server:
-- Host 1 with the infrastructure agent installed needs an outbound rule for port `1433`.
-- Host 2 with SQL Server needs an inbound rule for port `1433`.
+## Find data [#find-data]
```

Comment:
@zuluecho9 this seems out of sequence here. We need to tell them how to install the agent and the package before they view the data, right? Since this is part of the package instructions, how about ditching this?